docs/develop/build-a-dApp/build-smart-contracts/EVM-extensions-precompiles.md

```diff
@@ -2,5 +2,39 @@
 sidebar_position: 3
 ---
 
-# EVM Extension (Precompiles)
+# EVM Extension (Precompiled Smart Contracts)
 
+## Overview
```

Comment:
Would explain the purpose of EVM extensions first, check the [spec](https://www.notion.so/evmos/Stateful-Precompiled-Contracts-094297d698de4653a9cc2bc13dfd66dd#40c7bc3e45d0457b801fcf8076401939)

Then explain that its implemented through precompiled contracts.

Reply:
Updated some components of the description, let me know what you think. It can also be further improved once we have the stateful precompiles